cost/flexera/msp/master_org_cost_policy_currency/master_org_cost_policy_currency.pt

```diff
@@ -579,35 +556,36 @@ script "js_previous_month_cost_request", type: "javascript" do
 
     // create a map to use in the request metric field.
     var cost_metric = {
-      "Unamortized Unblended":"cost_nonamortized_unblended_adj",
-      "Amortized Unblended":"cost_amortized_unblended_adj",
+      "Unamortized Unblended": "cost_nonamortized_unblended_adj",
+      "Amortized Unblended": "cost_amortized_unblended_adj",
       "Unamortized Blended": "cost_nonamortized_blended_adj",
-      "Amortized Blended":"cost_amortized_blended_adj"
+      "Amortized Blended": "cost_amortized_blended_adj"
     }
+
     var start_at = "";
     var end_at = "";
     var now = new Date();
     // start_at from beginning of last month
     start_at = getFormattedDailyDate(new Date(now.getFullYear(), now.getMonth() - 1, 1))
     // end_at to end of last month
-    end_at = getFormattedDailyDate(addDays(new Date(now.setDate(0)),1))
+    end_at = getFormattedDailyDate(addDays(new Date(now.setDate(0)), 1))
 
     var billing_center_ids = []
     var org_billing_centers = _.filter(ds_billing_centers, function(bc){ return bc.org_id == rs_org_id })
     var top_billing_centers = _.reject(org_billing_centers, function(bc){ return bc.parent_id != null });
     billing_center_ids = _.map(top_billing_centers, function(value, key){ return value.id });
 
     var body = {
-      "dimensions":[]
-      "granularity":"day",
-      "start_at": start_at ,
+      "dimensions": []
```

Comment:
@XOmniverse - clearly didn't break anything before but should there be a comma at the end of this line?
```suggestion
      "dimensions": [],
```